wrappers/common/common.go

```diff
@@ -14,6 +14,10 @@ import (
 	libhoney "github.com/honeycombio/libhoney-go"
 )
 
+var (
+	ExtractCustomHeaders []string
```

Comment:
Please don't introduce global state like this. You could instead introduce a new function that accepts this string slice as a parameter in addition to a `*http.Request`, pass the latter to the existing `GetRequestProps` function, capture the returned map, and add to that map by consulting your string slice before returning it—all in your own code. There's no need to touch `GetRequestProps` to do this.